src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/auth/MontaguAuthenticationTests.kt

```diff
@@ -30,4 +30,22 @@ class MontaguAuthenticationTests : IntegrationTest()
 
         Assertions.assertThat(response.statusCode).isEqualTo(200)
     }
+
+    @Test
+    fun `user can logout`()
+    {
+        //login
+        val loginResponse = RequestHelper().getWithMontaguCookie("/login")
+
+        //logout
+        val cookie = loginResponse.headers["Set-Cookie"]
+        khttp.get("${RequestHelper().webBaseUrl}/logout",
+                headers = mapOf("Cookie" to cookie!!),
+                allowRedirects = false)
+
+        //after logout, user should be redirected when attempt to access base url
+        val response = khttp.get(RequestHelper()
+                .webBaseUrl, allowRedirects = false)
```

Comment:
But in any case if you don't provide the session cookie they will be redirected, so this doesn't verify that they're logged out!